README.md

```diff
@@ -1068,6 +1068,8 @@ Terminology differs somewhat between various database systems; please refer to t
 
 ## Limitations
 
+SQL 2017 detection support has been added. This support is limited to functionality already present for other versions. No new SQL 2017 specific functionality has been added in this release.
```

Comment:
We don't call it out here and I am not sure if we should, but we also don't make any particular effort to put up any safety rails for deprecated or altered features.

Reply:
We agreed that its fine as is, as we have upcoming work adding these features